### PR TITLE
Update Status of a Token Operation in Nexus

### PR DESCRIPTION
**Pull request by mohsinr**
In Nexus for any operation, we can change its `status `with `comment` this PR implements that API update in this SDK.

### CODE REVIEW

File: Nexus.Sdk.Token/Responses/TokenOperationResponse.cs [hunk 1]
```diff
@@ -77,4 +77,13 @@ public TokenOperationResponse(string code, string hash, OperationAccountResponse
         [JsonPropertyName("blockchainTransactionId")]
         public string BlockchainTransactionId { get; set; }
     }
+
+    public class UpdateOperationStatusRequest
```
Comment (nagarwal4) [inline Nexus.Sdk.Token/Responses/TokenOperationResponse.cs:81]: Why is this in the Response class?

Comment (nagarwal4) [reply Nexus.Sdk.Token/Responses/TokenOperationResponse.cs:81]: Please move it to Requests

**Review by nagarwal4 — CHANGES_REQUESTED** [review]

File: Nexus.Sdk.Token.Tests/Helpers/MockTokenServerProvider.cs [hunk 2]
```diff
@@ -331,5 +331,9 @@ public Task<NexusResponse> DeleteAccount(string accountCode)
         {
             throw new NotImplementedException();
         }
+        public Task<TokenOperationResponse> UpdateOperationStatusAsync(string operationCode, string status, string? comment = null, string? customerIPAddress = null)
```
Comment (nagarwal4) [inline Nexus.Sdk.Token.Tests/Helpers/MockTokenServerProvider.cs:334]:
```suggestion
        
        public Task<TokenOperationResponse> UpdateOperationStatusAsync(string operationCode, string status, string? comment = null, string? customerIPAddress = null)
```

File: Nexus.Sdk.Token/Responses/TokenOperationResponse.cs (hunk 1 above)
Comment (mohsinr) [reply Nexus.Sdk.Token/Responses/TokenOperationResponse.cs]: [Refactor code to move request code to requests file](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/55/commits/36675aff91474e96bc1c66e9dddb3605baa42928)

File: Nexus.Sdk.Token/Responses/TokenOperationResponse.cs [hunk 3]
```diff
@@ -77,4 +77,13 @@ public TokenOperationResponse(string code, string hash, OperationAccountResponse
...
+    {
+        [JsonPropertyName("status")]
+        public string Status { get; set; } = null!;
```
[the remark this answers is not in the document]
Comment (mohsinr) [reply Nexus.Sdk.Token/Responses/TokenOperationResponse.cs]: [Make status required and remove null value](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/55/commits/db11dd9fd971a590de7d7931a28b4b8ff36feffc)

**Pull request merged**